### PR TITLE
Fix for error thrown when mlflow logged metrics is a list.

### PR DESCRIPTION
**Pull request by arun-rajora**
Fix for error thrown when mlflow logged metrics is a list.

### CODE REVIEW

File: assets/aml-benchmark/components/src/benchmark_result_aggregator/main.py [hunk 1]
```diff
@@ -209,8 +209,9 @@ def _get_pipeline_params() -> Tuple[
 def _log_params_and_metrics(parameters: Dict[str, Any], metrics: Dict[str, Any]) -> None:
```
Comment (iamrk04) [inline assets/aml-benchmark/components/src/benchmark_result_aggregator/main.py:209]: Can we bump the version [over here](https://github.com/Azure/azureml-assets/blob/1cea65e5370e83214380d468fed6992db2d9f2f6/assets/aml-benchmark/components/benchmark-result-aggregator/spec.yaml#L7) to `0.0.2`?

**Review by iamrk04 — CHANGES_REQUESTED** [review]

File: assets/aml-benchmark/components/src/benchmark_result_aggregator/main.py (hunk 1 above)
Comment (arun-rajora) [reply assets/aml-benchmark/components/src/benchmark_result_aggregator/main.py:209]: Updated. Thanks!

File: assets/aml-benchmark/components/src/benchmark_result_aggregator/main.py [hunk 2]
```diff
@@ -209,8 +209,9 @@ def _get_pipeline_params() -> Tuple[
...
     """Log mlflow params and metrics to current run and parent run."""
     filtered_metrics = {}
+    allowed_metrics_type = [int, float, bool]
```
[the remark this answers is not in the document]
Comment (arun-rajora) [reply assets/aml-benchmark/components/src/benchmark_result_aggregator/main.py:212]: Bool gets type-casted and seems to work.

[the remark this answers is not in the document]
Comment (arun-rajora) [reply assets/aml-benchmark/components/src/benchmark_result_aggregator/main.py]: Updated to cast bool to float explicitly.

**Review by iamrk04 — APPROVED** [review]

**Review by iamrk04 — APPROVED** [review]

**Pull request merged**